packages/react-app/src/components/footer.tsx

```diff
@@ -34,55 +45,97 @@ const FooterLinkStyled = styled(Link)`
   margin-right: ${3 * GU}px;
   display: flex;
   align-items: center;
-  justify-content: space-between;
   text-decoration: none;
 `;
 
+const LineStyled = styled.div`
+  display: flex;
+  align-items: top;
+  max-width: 100%;
+  margin-top: ${GUpx(3)};
+`;
+
+const FooterNavItemStyled = styled(Link)`
+  color: #F7F7CE !important
```

Comment:
Have a look at dark-theme.ts and pick your theme color. It will likely not be this exact color when light theme

Reply:
Will remove color: #F7F7CE !important because it does not affect the link's color